File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        var i = 0;
        for(int n: nums){
            if(i==0 || n > nums[i - 1]){
                nums[i++] = n;
            }
        }
        return i;
    }
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count = 1
        l, k = 1, 1
        for r in range(1, len(nums)):
            if nums[r] == nums[r - 1]:
                count += 1
            else:
                count = 1
                
            if count <= k:
                nums[l] = nums[r]
                l += 1
                
        return l
                
        